File: 260123/제곱하여 출력하기/print-square-of-elements.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        // Please write your code here.
        Scanner sc=new Scanner(System.in);

        int N=sc.nextInt();
        double[] N_Score=new double[N];
        int[] answer=new int[N];

        for(int i=0; i<N; i++){
            N_Score[i]=sc.nextInt();
        }

        for(int i=0; i<N; i++){
            answer[i]=(int)Math.pow(N_Score[i],2);
            System.out.printf("%d ", answer[i]);
        }
        
    }
}

File: 260326/제곱하여 출력하기/print-square-of-elements.cpp
#include <iostream>
using namespace std;

int square(int x, int square){
    x*=x;
    return x;
}

int main() {
    // Please write your code here.
    int N;
    cin>>N;
    int arr[N];

    for(int i=0; i<N; i++){
        int num;
        cin>>num;
        arr[i]=square(num, N);

        cout<<arr[i]<<" ";
    }
    return 0;
}